Build failure with boost 1.76

When I try to build the project I get the following error:
```
[59/67] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -Dsymengine_EXPORTS -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/symengine/src/d972ac33b2-ad87f29739.clean -I. -isystem /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -std=c++11 -fPIC -fPIC -Wall -Wextra -Wno-unused-parameter -g -ggdb -Wno-unknown-pragmas -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -Wconversion -Wno-sign-conversion -Qunused-arguments -include /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/symengine/x64-osx-dbg/symengine/cotire/symengine_CXX_prefix.hxx -MD -MT symengine/CMakeFiles/symengine.dir/mp_boost.cpp.o -MF symengine/CMakeFiles/symengine.dir/mp_boost.cpp.o.d -o symengine/CMakeFiles/symengine.dir/mp_boost.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/symengine/src/d972ac33b2-ad87f29739.clean/symengine/mp_boost.cpp
FAILED: symengine/CMakeFiles/symengine.dir/mp_boost.cpp.o 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -Dsymengine_EXPORTS -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/symengine/src/d972ac33b2-ad87f29739.clean -I. -isystem /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -std=c++11 -fPIC -fPIC -Wall -Wextra -Wno-unused-parameter -g -ggdb -Wno-unknown-pragmas -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk -Wconversion -Wno-sign-conversion -Qunused-arguments -include /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/symengine/x64-osx-dbg/symengine/cotire/symengine_CXX_prefix.hxx -MD -MT symengine/CMakeFiles/symengine.dir/mp_boost.cpp.o -MF symengine/CMakeFiles/symengine.dir/mp_boost.cpp.o.d -o symengine/CMakeFiles/symengine.dir/mp_boost.cpp.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/symengine/src/d972ac33b2-ad87f29739.clean/symengine/mp_boost.cpp
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/symengine/src/d972ac33b2-ad87f29739.clean/symengine/mp_boost.cpp:282:12: error: no matching function for call to 'find_lsb'
    return find_lsb(i, int_<0>());
           ^~~~~~~~
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/boost/multiprecision/detail/bitscan.hpp:45:42: note: candidate function not viable: no known conversion from 'int_<0>' to 'const std::integral_constant<int, 0>' for 2nd argument
inline BOOST_MP_CXX14_CONSTEXPR unsigned find_lsb(Unsigned mask, const std::integral_constant<int, 0>&)
                                         ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/boost/multiprecision/detail/bitscan.hpp:146:28: note: candidate function not viable: no known conversion from 'const SymEngine::integer_class' (aka 'const number<boost::multiprecision::cpp_int_backend<>, boost::multiprecision::et_off>') to 'unsigned int' for 1st argument
BOOST_FORCEINLINE unsigned find_lsb(unsigned mask, std::integral_constant<int, 1> const&)
                           ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/boost/multiprecision/detail/bitscan.hpp:150:28: note: candidate function not viable: no known conversion from 'const SymEngine::integer_class' (aka 'const number<boost::multiprecision::cpp_int_backend<>, boost::multiprecision::et_off>') to 'unsigned long' for 1st argument
BOOST_FORCEINLINE unsigned find_lsb(unsigned long mask, std::integral_constant<int, 2> const&)
                           ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/boost/multiprecision/detail/bitscan.hpp:154:28: note: candidate function not viable: no known conversion from 'const SymEngine::integer_class' (aka 'const number<boost::multiprecision::cpp_int_backend<>, boost::multiprecision::et_off>') to 'boost::ulong_long_type' (aka 'unsigned long long') for 1st argument
BOOST_FORCEINLINE unsigned find_lsb(boost::ulong_long_type mask, std::integral_constant<int, 3> const&)
                           ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/boost/multiprecision/detail/bitscan.hpp:192:28: note: candidate function not viable: no known conversion from 'const SymEngine::integer_class' (aka 'const number<boost::multiprecision::cpp_int_backend<>, boost::multiprecision::et_off>') to 'boost::multiprecision::detail::uint128_type' (aka 'unsigned __int128') for 1st argument
BOOST_FORCEINLINE unsigned find_lsb(uint128_type mask, std::integral_constant<int, 0> const&)
                           ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/boost/multiprecision/detail/bitscan.hpp:213:53: note: candidate function not viable: requires single argument 'mask', but 2 arguments were provided
BOOST_FORCEINLINE BOOST_MP_CXX14_CONSTEXPR unsigned find_lsb(Unsigned mask)
                                                    ^
1 error generated.
```

--- a/symengine/mp_boost.cpp
+++ b/symengine/mp_boost.cpp
@@ -7,7 +7,6 @@
 #include <symengine/symengine_exception.h>
 #include <symengine/prime_sieve.h>
 
-using boost::mpl::int_;
 using boost::multiprecision::denominator;
 using boost::multiprecision::miller_rabin_test;
 using boost::multiprecision::numerator;

--- a/symengine/mp_boost.cpp
+++ b/symengine/mp_boost.cpp
@@ -279,7 +279,7 @@ unsigned long mp_scan1(const integer_class &i)
     if (i == 0) {
         return ULONG_MAX;
     }
-    return find_lsb(i, int_<0>());
+    return find_lsb(i, {});
 }
 
 // define simple 2x2 matrix with exponentiation by repeated squaring

